src/Deprecated12/ClassOrganization.extension.st

```diff
@@ -10,8 +10,8 @@ ClassOrganization >> addCategory: aString [
 { #category : #'*Deprecated12' }
 ClassOrganization >> allCategories [
 
-	self deprecated: 'Use #allProtocolsNames instead.' transformWith: '`@rcv allCategories' -> '`@rcv allProtocolsNames'.
-	^ self allProtocolsNames
+	self deprecated: 'Use #protocolsNames instead.' transformWith: '`@rcv allCategories' -> '`@rcv protocolsNames'.
```

Comment:
protocolsNames should be protocolNames, I think

Reply:
Indeed! I'm fixing that :)